src/views/layout.html

```diff
@@ -1,23 +1,23 @@
-{% extends "govuk/template.njk" %}
+{% extends "dist/govuk/template.njk" %}
 
-{% from "govuk/components/footer/macro.njk"                 import govukFooter %}
-{% from "govuk/components/header/macro.njk"                 import govukHeader %}
-{% from "govuk/components/error-summary/macro.njk"          import govukErrorSummary %}
-{% from "govuk/components/panel/macro.njk"                  import govukPanel %}
-{% from "govuk/components/input/macro.njk"                  import govukInput %}
-{% from "govuk/components/fieldset/macro.njk"               import govukFieldset %}
-{% from "govuk/components/button/macro.njk"                 import govukButton %}
-{% from "govuk/components/back-link/macro.njk"              import govukBackLink %}
-{% from "govuk/components/radios/macro.njk"                 import govukRadios %}
-{% from "govuk/components/date-input/macro.njk"             import govukDateInput %}
-{% from "govuk/components/textarea/macro.njk"               import govukTextarea %}
-{% from "govuk/components/radios/macro.njk"                 import govukRadios %}
-{% from "govuk/components/inset-text/macro.njk"             import govukInsetText %}
-{% from "govuk/components/summary-list/macro.njk"           import govukSummaryList %}
-{% from "govuk/components/select/macro.njk"                 import govukSelect %}
-{% from "govuk/components/cookie-banner/macro.njk"          import govukCookieBanner %}
-{% from "govuk/components/phase-banner/macro.njk"           import govukPhaseBanner %}
-{% from "govuk/components/notification-banner/macro.njk"    import govukNotificationBanner %}
+{% from "dist/govuk/components/footer/macro.njk"                 import govukFooter %}
```

Comment:
I would suggest to update the `nunjucks` configuration  file. On line 11, [here](https://github.com/cabinetoffice/github-requests-app/blob/56a9fff70b7f4ede51acaa82dfddc0fa54a4fd5f/src/config/nunjucks.ts#L11), change the component path to  `'node_modules/govuk-frontend/dist/'`, instead of changing it directly. Furthermore update the body end scripts with the right `initAll()` [here](https://github.com/cabinetoffice/github-requests-app/blob/56a9fff70b7f4ede51acaa82dfddc0fa54a4fd5f/src/views/layout.html#L91).

Reply:
Good spot, will do 👍

Reply:
New version of `govuk-frontend` now uses ESM modules for scripts - see for implementation example - https://frontend.design-system.service.gov.uk/installing-with-npm/#get-the-javascript-working

Reply:
Hi @harley-harris yes I am aware of the new implementation. Do you want me to have a look at it?

Reply:
No, just adding note in PR for visibility, I wasn't aware it changed 👍